shared/store/transaction_queue.ts

```diff
@@ -6,6 +6,8 @@ export interface DfTxSigner {
   sign: (account: WhaleWalletAccount) => Promise<CTransactionSegWit>;
   title?: string;
   description?: string;
+  loadingMessage?: string;
```

Comment:
do we still need this after the revert?

Reply:
Yes, this is for password verification:
<img width="643" alt="image" src="https://user-images.githubusercontent.com/17063281/192198712-372fd588-2e06-4370-bd8a-4b699870d81a.png">

Currently the message is "Signing your transaction" --> should we keep this?